CHANGELOG.md

```diff
@@ -8,6 +8,11 @@
 * Optional parameters are now typed as `T | undefined | null` to reflect the actual JS behavior.
   [#4188](https://github.com/rustwasm/wasm-bindgen/pull/4188)
 
+### Fixed
+
+- Fixed JS keyword-like identifiers not being handled correctly. Keywords on imports will no longer be escaped. Using keyword identifiers in places that caused invalid JS code gen will now error at compile time.
```

Comment:
This needs to be detailed. A list of all the things that are now a compile error is the minimum. Instead of listing all the keywords here link to it. But "in places" needs to be expanded in detail.

Reply:
Done.